test/tools/libtesteth/ImportTest.h

```diff
@@ -84,8 +84,9 @@ class ImportTest
 	using StateAndMap = std::pair<eth::State, eth::AccountMaskMap>;
 	using TrExpectSection = std::pair<transactionToExecute, StateAndMap>;
 	bool checkGeneralTestSectionSearch(json_spirit::mObject const& _expects, std::vector<size_t>& _errorTransactions, std::string const& _network = "", TrExpectSection* _search = NULL) const;
+    void makeBlockchainTestFromStateTest(std::vector<eth::Network> const& _networks) const;
```

Comment:
Can you change the signature so that the input goes as an argument and the output comes as a return value?

Or, will you document where the input is taken from, and where the output goes?

Reply:
well I wanted it to be out of header. but it is too much linked with private fields

Reply:
Inside or outside of the header, a comment is needed about what this function does.

Reply:
what do you want? just a comment?

Reply:
this is a small PR to split a large function. I am planing to make a new future to testeth soon.

Reply:
if we spend another week to just split one function into two I wont be able to make new changes in a month.

Reply:
@winsvega yes, I want a comment.

You can say "I'll do xyz later".  I'll make a note.

Reply:
I guess after Pwaels review ther would have to be done like lots of changes. here.

Reply:
A comment here is the only remaining issue for me.